spark-gpu/rapids.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+readonly ROLE=$(/usr/share/google/get_metadata_value attributes/dataproc-role)
+readonly LINUX_DIST=$(/usr/share/google/get_metadata_value attributes/linux-dist)
+
+readonly DEAFULT_INIT_ACTIONS_REPO=gs://dongm-gcp-shared
+readonly INIT_ACTIONS_REPO="$(/usr/share/google/get_metadata_value attributes/INIT_ACTIONS_REPO ||
+  echo ${DEAFULT_INIT_ACTIONS_REPO})"
+
+echo "Cloning RAPIDS initialization action from '${INIT_ACTIONS_REPO}' ..."
+RAPIDS_INIT_ACTION_DIR=$(mktemp -d -t rapids-init-action-XXXX)
+readonly RAPIDS_INIT_ACTION_DIR
+gsutil -m rsync -r "${INIT_ACTIONS_REPO}/spark-gpu" "${RAPIDS_INIT_ACTION_DIR}"
+
+if [[ "${LINUX_DIST}" == 'ubuntu' ]]; then
+  mv "${RAPIDS_INIT_ACTION_DIR}/internal/install-gpu-driver-ubuntu.sh" "${RAPIDS_INIT_ACTION_DIR}/internal/install-gpu-driver.sh"
```

Comment:
Is it possible to reuse [GPU init action](https://github.com/GoogleCloudPlatform/dataproc-initialization-actions/tree/master/gpu) instead of custom GPU drivers install scripts?

Reply:
There are a few additional dependencies required like NCCL and rapids spark libraries. Also updated install-gpu-driver-ubuntu.sh is NVIDIA's recommended route to install Tesla drivers.

Reply:
I have added support for NVIDIA-provided GPU drivers and CUDA to GPU init action. Please update this PR to use it.

Reply:
Done.